flashcards/web/package.json

```diff
@@ -62,12 +63,14 @@
         "eslint-plugin-react-hooks": "^4.6.0",
         "eslint-plugin-simple-import-sort": "^10.0.0",
         "happy-dom": "^8.9.0",
+        "json-server": "^0.17.3",
         "postcss": "^8.4.21",
         "prettier": "^2.8.4",
         "tailwindcss": "^3.2.7",
         "typescript": "^4.9.3",
         "vite": "^4.1.0",
         "vite-eslint-plugin": "^1.9.2",
         "vite-tsconfig-paths": "^4.0.5"
-    }
+    },
+    "proxy": "http://0.0.0.0:3001"
```

Comment:
I'd just expose it directly from docker-compose.yaml not via vite

Reply:
I removed proxy